build-logic/documentation/src/main/groovy/gradlebuild/docs/GradleKotlinDslReferencePlugin.java

```diff
@@ -131,7 +131,7 @@ private static void configureSourceLinks(Project project, GradleDocumentationExt
     private static URI toUri(File projectRootDir, File file, String commitId) {
         try {
             Path relativeLocation = projectRootDir.toPath().relativize(file.toPath());
```

Comment:
🤔 Could this instead use `URI.relativize()` or `URL.relativize()` to produce the proper relative path always with slashes for the line below instead?

Reply:
Ah, good idea, thy.
I give it a try, should probably work.

Reply:
Yes, works fine. Update is pushed.